docker/entrypoint.sh

```diff
@@ -6,16 +6,17 @@
 set -euo pipefail
 
 # Keep these in sync with DATABASE_URL.
-DATABASE_HOST='mysql'
+DATABASE_HOST=${DATABASE_HOST:-mysql}
```

Comment:
This made sense when I was trying to make sure MySql was running on Travis before trying to run tests.